src/ProductView/TabBar/QueryInterface/QueryInterface.js

```diff
@@ -28,8 +29,10 @@ class QueryInterface extends Component {
                 }
             ],
             eventQuery: '',
```

Comment:
Please do not add eventQuery to state. This makes no sense in this case but causes unnecessary rerenderings.
Better add it to this.